src/features/ajax/aggregate/index.js

```diff
@@ -15,251 +14,183 @@ import { SUPPORTABILITY_METRIC_CHANNEL } from '../../metrics/constants'
 import { AggregateBase } from '../../utils/aggregate-base'
 import { parseGQL } from './gql'
 import { getNREUMInitializedAgent } from '../../../common/window/nreum'
+import Chunk from './chunk'
 
 export class Aggregate extends AggregateBase {
   static featureName = FEATURE_NAME
+  #agentRuntime
+  #beacon
+  #proxyBeacon
```

Comment:
Curious why the need to do this.  This is already stored in the runtime obj, so abstacting these to another var (esp a private one), is just code duplication

Reply:
private methods in general just add extra overhead (esp. when paired with babel) and make things harder to test in general anyway (since now you cant just reference the property directly in a test), so we should be considerate about when and where they get used, is my only ask

Reply:
I could change these to be info, init, and runtime, but those references (instead of value) will still be stored as variables. Is that better?

Reply:
Done